vignettes/sf1.Rmd

```diff
@@ -128,7 +128,7 @@ Note that `CIRCULASTRING`, `COMPOUNDCURVE` and `CURVEPOLYGON` are not described
 ## Coordinate reference system
 
 Coordinates can only be placed on the Earth's surface when their coordinate
-reference system (CRS) is known; this may be a spheroid CRS such as WGS84, a projected, two-dimensional (Cartesian) CRS such as a UTM zone or Web Mercator, or a CRS
+reference system (CRS) is known; this may be a spheroid CRS such as [WGS84](https://en.wikipedia.org/wiki/World_Geodetic_Syste), a projected, two-dimensional (Cartesian) CRS such as a UTM zone or Web Mercator, or a CRS
```

Comment:
this link is wrong

Reply:
I will replace it with: https://gisgeography.com/wgs84-world-geodetic-system/